data/language/en-GB.yml

```diff
@@ -2202,7 +2202,7 @@ strings:
   2147: "Every {INT32} months"
   2148: "{COLOUR WINDOW_2}Generator:"
   2149: Original
-  2150: Improved
+  2150: Simplex
```

Comment:
These terms are fine internally, but might not mean much to the average player. I wonder if we should change them, e.g.:

```yaml
  2149: Locomotion map generator
  2150: Simplex noise generator
```